features/in_onderzoek.feature

```diff
@@ -20,31 +20,27 @@ Functionaliteit: in onderzoek
-      | 1         | 555550001                   | Arnitta           |                     | 19231213              | <waarde>                        | 20120920                       |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende oudergegevens in de registratie
-      | Categorie | Voornamen (02.10) | Geslachtsaanduiding (04.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 2         | Noa               | V                           |                                 |                                |                               |
-      | 3         | Jan               | M                           |                                 |                                |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende partnergegevens in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen | Geslachtsaanduiding (04.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 5         | 555550002                   | Karel     | M                           | 20091102              |                          |                                 |                                |                               |
+      Gegeven de persoon met burgerservicenummer '555550001' heeft de volgende gegevens
+      | voornamen (02.10) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | 
+      | Arnitta           | 19231213              | <waarde>                        | 20120920                       |
+      En de persoon heeft een ouder '1' met de volgende gegevens
+      | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | Noa               | V                           |
+      En de persoon heeft een ouder '2' met de volgende gegevens
+      | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | Jan               | M                           |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) | voornamen (02.10 | geslachtsaanduiding (04.10) | datum aangaan (06.10) |
+      | 555550002                   | Karel            | M                           | 20091102              |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | ouders,partners,<fields>        |
-      Dan heeft de persoon de volgende 'persoonInOnderzoek' gegevens
+      Dan heeft de response een persoon met de volgende 'persoonInOnderzoek' gegevens
```

Comment:
Bij een end-to-end test wordt de response gevalideerd van de proxy. Hier wordt de response gevalideerd van de GBA API